packages/kbn-monaco/src/esql/lib/ast/autocomplete/factories.ts

```diff
@@ -50,7 +50,7 @@ export function getAutocompleteFunctionDefinition(fn: FunctionDefinition) {
 }
 
 export function getAutocompleteBuiltinDefinition(fn: FunctionDefinition) {
-  const hasArgs = fn.signatures.some(({ params }) => params.length);
+  const hasArgs = fn.signatures.some(({ params }) => params.length > 1);
```

Comment:
Is this change just for clarity?

Reply:
No, this is actually catching a bug for those builtin functions with a single argument.
In that case it should not complete those with the second argument.